src/components/dockerRegistry/Docker.tsx

```diff
@@ -84,22 +84,27 @@ function CollapsedList({
     ...rest
 }) {
     const [collapsed, toggleCollapse] = useState(true);
+
+    const setToggleCollapse = () => {
+        toggleCollapse(false)
+    }
+    
     return (
         <article className={`collapsed-list collapsed-list--docker collapsed-list--${id ? 'update' : 'create dashed'}`}>
-            <List onClick={(e) => toggleCollapse((t) => !t)}>
+            <List onClick={setToggleCollapse} className={`${!id && !collapsed ? 'no-grid-column':''}`}>
                 {id ? (
                     <List.Logo>
-                        {' '}
                         <div className={'registry-icon ' + registryType}></div>
                     </List.Logo>
-                ) : (
+                ) : (collapsed &&
                     <List.Logo>
                         <Add className="icon-dim-24 fcb-5 vertical-align-middle" />
                     </List.Logo>
                 )}
```

Comment:
We can this condition now to something like
```
{id && <List.Logo>}
{!id && collapsed && <List.Logo>}
```

Reply:
done